### PR TITLE
metrics updates

### PR DESCRIPTION
**Pull request by snopoke**
* record request counts for auth_keys endpoint
* record form counts for all form submission requests
* a bit of refactoring

review by commit :fish:

### CODE REVIEW

File: corehq/apps/receiverwrapper/views.py [hunk 1]
```diff
@@ -41,12 +41,26 @@
 
 def _process_form(request, domain, app_id, user_id, authenticated,
                   auth_cls=AuthContext):
+    metic_tags = [
```
Comment (gcapalbo) [inline corehq/apps/receiverwrapper/views.py]: not blocking but did you mean `metric_tags`

**Review by gcapalbo — APPROVED** [review]

File: corehq/apps/receiverwrapper/views.py (hunk 1 above)
Comment (snopoke) [inline corehq/apps/receiverwrapper/views.py]: 891b8ac

File: corehq/apps/mobile_auth/views.py [hunk 2]
```diff
@@ -60,6 +62,7 @@ def get_payload(self):
 
 @login_or_digest_or_basic_or_apikey()
 @require_GET
+@count_by_response_code('commcare.auth_keys.fetch.count')
```
Comment (benrudolph) [inline corehq/apps/mobile_auth/views.py]: can we get this metric from `nginx.requests` by filtering for a url?

Comment (snopoke) [inline corehq/apps/mobile_auth/views.py]: we can, though we may have to get rid of the url tagging if we don't want to pay for custom metrics

**Pull request merged**